Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -8,8 +8,17 @@ import QuartzCore
 class LayoutManager: NSLayoutManager {
 
     /// Blockquote's Left Border Color
-    ///
-    var blockquoteBorderColor: UIColor? = UIColor(red: 0.52, green: 0.65, blue: 0.73, alpha: 1.0)
+    /// Include 3 nested values
+    
+    var blockquoteBorderColor = [0 : UIColor(hexString: "9252FF")!,
```

Comment:
Why couldn't be this a simple array like this:
```
var blockquoteBorderColor: [UIColor] = [UIColor(hexString: "9252FF")!, UIColor(hexString: "5A4AE8")!, UIColor(hexString: "5D77FE")!, UIColor(hexString: "4A8EE8")!, UIColor(hexString: "52CAFF")!]

Reply:
hmm, now that you mention it, I guess this could work. For some reason at the time I thought I needed a dictionary to ensure I kept the order / accessed the right color at the right level. Let me investigate and make the change.